modules/vaos/spec/request/appointment_requests_request_spec.rb

```diff
@@ -10,6 +10,8 @@
     allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
   end
 
+  let(:inflection_header) { { 'X-Key-Inflection' => 'camel' } }
```

Comment:
Looks like on line 95 you could use this rather variable instead of writing it out, possibly other places.